lesson02/main.tf

```diff
@@ -9,3 +9,4 @@ terraform {
     region = "us-east-2"
   }
 }
+
```

Comment:
styling: let's remove the extra line

Reply:
we can do this in the next cycle this is not a blocker, thanks for pointing that out

Reply:
This will be done in another PR. Created an issue to keep track of this #33